ambari-web/app/controllers/main/admin/highAvailability/nameNode/rollbackHA/step2_controller.js

```diff
@@ -27,7 +27,8 @@ App.RollbackHighAvailabilityWizardStep2Controller = App.HighAvailabilityWizardSt
       name: 'admin.high_availability.getNnCheckPointStatus',
       sender: this,
       data: {
-        hostName: hostName
+        hostName: hostName,
+        nameNodeId: App.HostComponent.find().findProperty('componentName', 'NAMENODE').get('compId')
```

Comment:
I would prefer to see these lookups performed outside of the object literal. That way you can check if they return a value before you proceed.

Reply:
There will always be an id associated with a component. If not then it will be a BE bug and would be easily identified with this console error.